docs/markdown/intro/README.md

```diff
@@ -126,45 +124,50 @@ npm install vue bootstrap-vue bootstrap
 yarn add vue bootstrap-vue bootstrap
 ```
 
-Then, register BootstrapVue in your app entry point:
+Then, register BootstrapVue in your app entry point (typically `app.js` or `main.js`):
 
 ```js
 // app.js
 import Vue from 'vue'
 import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'
 
-// Install BootstrapVue
+// Tell Vue to make BootstrapVue available throughout your project
 Vue.use(BootstrapVue)
 // Optionally install the BootstrapVue icon components plugin
 Vue.use(IconsPlugin)
 ```
 
-And import Bootstrap and BootstrapVue `css` files:
+You must also import the `css` files. This consists of 2 or 3 files in this specific order: your own custom `scss` file (for example if you are globally redefining the 'primary' color) and the Bootstrap and BootstrapVue `css` files. There are two ways of doing this.
```

Comment:
> (for example if you are globally redefining the 'primary' color)
I'm not sure I understand this section.
If you're importing `Bootstrap.css`, then it wont matter what changes you've made in your own `scss` file, as it wont compile bootstrap with it. Or am I misunderstanding it?

Reply:
When importing the Bootstrap *CSS* `dist` files, there is no way to change/redefine variables.
This is only possible when using the SCSS files.

Reply:
The important part is already quite well described:

> Be sure to @import or define your custom variable values before including Bootstrap SCSS (bootstrap.scss), and include BootstrapVue SCSS (bootstrap-vue.scss) after that to ensure variables are set up correctly.

Reply:
Ah I missed that subtlety, sorry! I was careless between css and scss. I will correct and come back to you.